src/components/Global/MoreInfo/index.tsx

```
@@ -1,32 +1,104 @@
+                    style={tooltipStyle}
+                    className="whitespace-pre-line rounded border border-n-1 bg-white px-3 py-2 shadow-lg"
                 >
-                    <Menu.Items className="absolute bottom-full right-0 z-30 mb-1 mr-1 w-64 border border-n-1 bg-white px-3 py-2 shadow-lg md:left-0 md:right-auto">
-                        <Menu.Item as={'label'} className={'text-h8 font-normal text-black'}>
-                            {text}
-                        </Menu.Item>
-                    </Menu.Items>
-                </Transition>
-            </>
+                    <div className="block text-h8 font-normal text-black">
+                        {html ? <div dangerouslySetInnerHTML={{ __html: text }} /> : <div>{text}</div>}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)